src/routes/source.js

```diff
@@ -25,7 +24,6 @@ async function invalidateCollab(api, url, env) {
 }
 
 export async function deleteSource({ req, env, daCtx }) {
-  await postObjectVersion(req, env, daCtx);
```

Comment:
IIUC this line here is to save a version of the file before deletion, so that it could be restored if the delete was accidental. I think it would be good to keep that.

/cc @karlpauls

Reply:
IIRC, there was a discussion about this (OOB). The problem was it didn't consider directories correctly. IMO it would make sense to keep it unless it causes big performance concerns but it would need to be only done for files and you'd have to evaluate the impact for large trees I guess.

Reply:
I specifically asked @auniverseaway about this, and he to get rid of it.

Reply:
This was discussed on Slack w/ @auniverseaway , and the decision was to remove it.